longest_palindromic_substring/longest_palindromic_substring.py

```diff
@@ -0,0 +1,46 @@
+def expand_around_center(test, left, right):
+    """
+    :param test: Input string whose substrings are to be checked
+    :param left: Left end of result palindromic substring
+    :param right: Right end of result palindromic substring
+    :return: Length of palindromic substring
+    """
+    n = len(test)
```

Comment:
Since using it only one time. I think no need to assign `len(test)` to a new variable use it directly.